8_reference/73_techniques_best_practices.txt

```diff
@@ -25,6 +25,18 @@
 
 ==== Writing convention
 
+===== Technique Naming Guidelines
+
+The following rules should be followed when naming a new Technique:
+
+   * Try to keep names as short as possible, to improve readability
+   * Read the exinsting technique list, and particularly techniques related to what you are writing. The new names should be consistent with existing ones.
+   * The name should be a nominal group (not "Manage file content" or "Set Service state" but "File content" or "Service state)". It describes the target of the action, not the action itself.
+   * The name should look like: General Concept (package, file, etc.) + Source (from file, etc.) + Implementation details (platform, software name, etc.)
```

Comment:
Examples of this would be useful too. Package repositories may be a good example, or variables?